components/service/glean/src/test/java/mozilla/components/service/glean/private/TimespanMetricTypeTest.kt

```diff
@@ -88,6 +89,36 @@ class TimespanMetricTypeTest {
         assertEquals(1, testGetNumRecordedErrors(metric, ErrorType.InvalidValue))
     }
 
+    @Test
+    fun `The API must correctly clear the timer state on stop()`() {
```

Comment:
IMO this test name doesn't accurately reflect what's happening.
It's more about not double-recording and tracking errors.

(Consider this a bikeshed only)

Reply:
True. While I agree with you, given the time sensitivity of this, I'd leave it as it is: this test only applies to ac and will be deprecated anyway.